docs/whirlpool/docs/05-More Resources/08-Interact via CPI.mdx

```diff
@@ -1,28 +1,11 @@
 import DocCard from '@theme/DocCard';
```

Comment:
Does this still need a separate page since it just refers to the same package

Reply:
Probably not. I will remove it.